be/src/exec/tablet_sink.cpp

```diff
@@ -1200,37 +1204,33 @@ Status OlapTableSink::_incremental_open_node_channel(const std::vector<TOlapTabl
                     replica->set_node_id(node_id);
                 }
 
-                tablets.emplace_back(std::move(tablet_info));
+                index_tablets_map[index.index_id].emplace_back(std::move(tablet_info));
```

Comment:
What's the difference between `index_tablets_map[index.index_id]` and `tablets`?

Reply:
each index has it's own tablets